CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
-# v 2.2.0 (?)
+# v 2.1.1 (?)
```

Comment:
`pytest-inmanta` uses the old changelog format. You should never update the version number on such a changelog. This is done by the release pipeline. Just adding the entry to the top-level section is sufficient.
```suggestion
# v 2.2.0 (?)
```